README.md

```diff
@@ -30,13 +30,19 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+Document object model- is an interface for html that allows the document to be manipulated
+
 2. What is an event?
+Events represent user actions that are executed through functions.
```

Comment:
the event object contains all the information that we need to track a user's action when they perform the event.